src/main/java/org/zalando/baigan/service/ChainedConfigurationRepository.java

```diff
@@ -51,6 +51,6 @@ public Optional<Configuration> get(@Nonnull String key) {
 
     @Override
     public void put(@Nonnull String key, @Nonnull String value) {
```

Comment:
A bit of irrelevant to the PR question, but what was the intention of this method? All implementations throw `UnsupportedOperationException` :thinking:

Reply:
I was wondering the same, but decided to ignore it for this PR. Looks like the results of a YAGNI case to me :grin:

Reply:
Original usages of baigan-config used the same library code to implement processes to update the configuration (repositories), i.e. the applications themselves were toggling off feature toggles when certain conditions were met.